setup.py

```diff
@@ -73,7 +73,7 @@
     #
     # This field corresponds to the "Home-Page" metadata field:
     # https://packaging.python.org/specifications/core-metadata/#home-page-optional
-    url='https://github.com/IBM/stix-shifter',  # Optional
+    url='https://github.com/opencybersecurityalliance/stix-shifter',  # Optional
```

Comment:
Should we also update this License stated at line 102?
`'License :: OSI Approved :: MIT License',`